thirdeye/thirdeye-frontend/app/helpers/manage-alert-utils.js

```diff
@@ -0,0 +1,226 @@
+import { helper } from '@ember/component/helper';
+import moment from 'moment';
+
+/**
+ * Handles types and defaults returned from eval/projected endpoints
+ * @param {Number|String} metric - number or string like 'NaN', 'Infinity'
+ * @param {Boolean} allowDecimal - should this metric be shown as whole number?
+ * @returns {Object}
+ */
+export function formatEvalMetric(metric, allowDecimal = false) {
+  let shown = 'N/A';
+  if (Ember.typeOf(metric) === 'number') {
+    if (allowDecimal) {
+      shown = (Number(metric) === 1 || Number(metric) === 0)
+        ? metric * 100
+        : (metric * 100).toFixed(1);
+    } else {
+      shown = metric;
+    }
+  }
+  return shown;
+}
+
+/**
+ * Pluralizes and formats the anomaly range duration string
+ * @param {Number} time
+ * @param {String} unit
+ * @returns {String}
+ */
+export function pluralizeTime(time, unit) {
+  const unitStr = time > 1 ? unit + 's' : unit;
+  return time ? time + ' ' + unitStr : '';
```

Comment:
What if time is 0? Is it supposed to default to an empty string in that case or show 0 units?

Reply:
Yes - good question. A bit further down, I'm only using "non-zero" values (discarding the rest). See `noZeroDurationArr`